### DISCUSSION TITLE
Translations 2nd revisions

### DISCUSSION THREAD

**Discussion by hammerdirt-fieldops** [hammerdirt-analyst/IQAASL-End-0f-Sampling-2021#23] [General]
2nd revisions

**Comment by hammerdirt-fieldops** [comment]
Executive summary in English and German remove from top
IQAASL era un progetto finanziato dalla Confederazione Svizzera

**Reply by hammerdirt-analyst** [reply]
ref 3d50b70

**Comment by hammerdirt-fieldops** [comment]
Seen und Flüsse
1.7.1. Schaumstoffe und Kunststoffe nach Größe
Change _Alle Erhebungsgebiete fragmented foams and plastics by size group._
to: Alle Erhebungsgebiete: fragmentierte Kunststoffe und geschäumte Kunststoffe nach Größe, Median p/100m, Anzahl der Stücke und Prozent der Gesamtmenge.
Change in English too: _All survey areas: fragmented plastics and foamed plastics by size, median p/100m, number of pieces, and percent of total.._

**Reply by hammerdirt-analyst** [reply]
ref 915a3fb

**Comment by hammerdirt-fieldops** [comment]
Die Alpen und der Jura
2.6.2. Landnutzungsprofil der Erhebungsorte
KeyError

**Reply by hammerdirt-analyst** [reply]
[main d3b28c8] i29 -- key error

**Comment by hammerdirt-fieldops** [comment]
Aare
3.3. Datenerhebungen Ergebnisse und Landnutzung
remove ** before table. Same for English version

**Reply by hammerdirt-analyst** [reply]
[main acdb29b] i27 -- markdown syntax

**Comment by hammerdirt-fieldops** [comment]
In Annex sections of report in english & duetsch remove organization section # and replace with: Survey locations and inventory
De: Die Erhebungsorte und das Inventar der Objekte

**Reply by hammerdirt-analyst** [reply]
ref commit: 44406c1f40e3cc71ee0870d7b1d4d948181095c0 i30 - german version

ref [main 8399918] i30 - english version and new build

**Comment by hammerdirt-fieldops** [comment]
Vorwort  Remove underscore before Paul Scherrer and add this link to French and Italian versions

Überwachung von Müll mit der nächsten Generation von Umweltingenieuren 2016-2021, Zusammenarbeit mit Bhavish Patel, [_Paul Scherrer Institute](https://www.psi.ch/en), Christian Ludwig, [Paul Scherer Institute](https://www.psi.ch/en)/[EPFL](https://www.epfl.ch/en/). [Repository](https://github.com/hammerdirt-analyst/swe)

**Reply by hammerdirt-analyst** [reply]
ref [main f42dedb] i28 -- links and formatting

**Comment by hammerdirt-fieldops** [comment]
Lake section de: Lac Léman = Genfersee

**Reply by hammerdirt-analyst** [reply]
This will not be changed unless the customer requests. Regional issue, for someone in the region the name of the lake is Lac Léman --  not Lake Geneva. Therefore its the german speakers who have adopted the wrong name. Most people who live in Switzerland and any French departments in the region know the right name

**Comment by hammerdirt-fieldops** [comment]
Vorwort de change Figure 1 to Abbildung 1

**Comment by hammerdirt-fieldops** [comment]
Section 21.3.4. Landnutzungsprofil
Change Spearman”s ρ orSpearmans to: Spearmans ρ oder Spearmans....
Remove English phrase: An association suggests that survey totals for that object will change in relation to the amount of space attributed to that feature, or in the case of roads or river intersections, the quantity. The magnitude of the relationship is not defined and any association is not linear.

**Reply by hammerdirt-analyst** [reply]
ref [main 7024783] i33 translation and punctuation

**Comment by hammerdirt-fieldops** [comment]
21.3.3. Land use profile: english version correct: _Spearman**”s ρ oor** Spearmans ranked_...

**Reply by hammerdirt-analyst** [reply]
ref [main 7024783] i33 translation and punctuation

**Comment by hammerdirt-fieldops** [comment]
add (pc ) parent code to acronyms

**Reply by hammerdirt-analyst** [reply]
ref [main f1d4bf9] i33 and new build

**Comment by hammerdirt-fieldops** [comment]
Executive summary, The most common objects section remove figure 2 in phrase: For example, **figure 2**, a total of 8,485 cigarette filters...
German version change to: So wurden beispielsweise in 87 % der Datenerhebungen insgesamt 8.485 Zigarettenfilter identifiziert, was 15 % der insgesamt gesammelten Gegenstände und einem Median von 20 Zigarettenfiltern pro 100 m Uferlinie entspricht.

Italian version: Per esempio, un totale di 8.485 filtri di sigarette sono stati identificati nell'87% delle raccolte di dati, che rappresentano il 15% del totale degli oggetti raccolti e una mediana di 20 filtri di sigarette per 100 m di costa.

French version: Par exemple, un total de 8 485 filtres de cigarettes ont été identifiés dans 87 % des collectes de données, ce qui représente 15 % du total des éléments collectés et une médiane de 20 filtres de cigarettes par 100 m de rivage.

**Reply by hammerdirt-analyst** [reply]
We will handle this on the next round of changes

**Comment by AmandaFinger** [comment]
I have some questions on chapter 20.3 "The data" (applies to English and German version):
Why does the caption of the map of survey locations indicate that you did 300 surveys at 84 different locations? According to the Executive Summary you did 386 surveys at 143 different locations. Why are those not the same numbers?

Please also check the enumartion in the German chapter, since 20.3 "The data" ist chapter 20.2 in the German chapter.

**Reply by hammerdirt-analyst** [reply]
In chapter 20 we are comparing like items under like conditions. So there are no "river surveys" in chapter 20. Furthermore the locations in Walensee were excluded because the land use data is incomplete for that region. So we could not really differentiate between urban and rural locations in that area.

The above is not explained adequately in the text. will open an issue and make that correction. We will need to relook at the german version to include that. Good catch!

**Reply by hammerdirt-fieldops** [reply]
20.3 The data
Map and title are fine. **Remove** "There were 300 surveys at 84 different locations."
**Change to** “Results of 300 surveys at 84 urban and rural locations from March 2020 - May 2021. Rivers and Walensee are not included.”

**Reply by hammerdirt-fieldops** [reply]
Please standardize the following citation:
**Note the title change**
Erismann, R., Erismann, S.
Identification, quantification and analysis of observable anthropogenic  litter along Swiss lake systems (IQAASL)
2021
doi: 10.5281/zenodo.6053925,
URL: https://hammerdirt-analyst.github.io/IQAASL-End-0f-Sampling-2021/intro.html.

**Reply by hammerdirt-fieldops** [reply]
20.3 The data  Results of 300 surveys at 84 urban and rural locations from March 2020 - May 2021. Rivers and Walensee are not included.
German: Ergebnisse von 300 Datenerhebungen an 84 städtischen und ländlichen Standorten von März 2020 bis Mai 2021. Flüsse und Walensee sind nicht enthalten.

**Reply by hammerdirt-analyst** [reply]
These items were handled with commit: d2dd5fb4a16fc2fba157d9c314d00cf9f73b8e63

**Comment by AmandaFinger** [comment]
General remark on the report and the charts: In Switzerland we do not use the symbol "ß" as a "double S" (Straßen, Straßenkilometer, Ausreißer, mengenmäßig). Please change it into "Strassen", "Strassenkilometer", "Ausreisser", "mengenmässig" etc. => instead of the symbol "ß" you add "ss".

General remark on the German charts: Make sure the charts and tables are consistently in German. 
 - For instance, chart in chapter 3 "Aare" is in English, 
 - For instance, the chart in chapter 3.1.2 is partly in English, like "Plastic-kg" (in German it would be: Plastik-kg, or even better: kg Plastik | Objekte instead of Objekten (spelling mistake)
 - For instance, chart in chapter 3.1.3: Aare-Erhebungsgebiet: monthly median (in German: Aare-Erhebungsgebiet: monatlicher Medianwert) 

The following sentences, which appear in chapter 3.1.1, 4.1.1, 5.1.1 and 6.1.1, are neither in English nor in German comprehensible. What are you trying to say with the following sentences:
German
Straßen werden als Gesamtzahl der Straßenkilometer im Umkreis von 1’500 m angegeben. Flussmündung ist ebenfalls eine ordinale Rangfolge der Anzahl der Flüsse/Kanäle, die einen See innerhalb von 1’500 m um den Erhebungsort herum durchqueren.

English
Streets are reported as the total number of kilometers of streets within the 1500m radius. Intersects is also an ordinal ranking of the number of rivers/canals that intersect a lake within 1500m of the survey location.

**Reply by hammerdirt-analyst** [reply]
This means exactly what it says, I don't know how to explain it better:

Streets are reported as the total number of kilometers of streets within the 1500m radius. Intersects is also an ordinal ranking of the number of rivers/canals that intersect a lake within 1500m of the survey location.

Other changes were made to the requested documents and any other applicable documents in #41

**Comment by AmandaFinger** [comment]
Chapters 3.1.3, 4.1.3, 5.1.3 and 6.1.3 in German: 
Rechts: Aare-Erhebungsgebiet empirische kumulative Verteilung der Datenerhebunge.
"Datenerhebunge**n**" - please add an "n".

Chapter 3.1.4, 4.1.4, 5.1.4 and 6.1.4 + same charts in alle the lake chapters (7-13):
English: Add the total number of objects in the table - this information seems to be missing (in German: "Totalobjekte")
German: "Totalobjekte" is not really comprehensible, please replace by "Abfallobjekte total"

Chapters 3.3, 4.3, 5.3, 6.3
Original sentence in English:
Below: Aare survey area ranked correlation of the most common objects with respect to land use profile. For all valid lake samples n=118.

Sentence in German (please **add**):
**Unten:** Ausgewertete Korrelation der am häufigsten gefundenen Objekte in Bezug auf das Landnutzungsprofil im Aare-Erhebungsgebiet. Für alle gültigen **Erhebungen an Seen** n=118.

Chapters 3.4, 4.4, 5.4, 6.4 (German)
Please correct the caption for the second chart as follows (example for chapter 3.4):
**Unten:** Verwendungszweck der gefundenen Objekte Median p/100m im Aare-Erhebungsgebiet. Fragmentierte Objekte, die nicht eindeutig identifiziert werden können, werden weiterhin nach ihrer Grösse klassifiziert.

Chapters 3.5, 4.5, 5.5 and 6.5 and also 3.5.1 etc. (German and English)
Why do you call the chart caption "Aare survey area rivers" if the chart also shows the lakes? Maybe you should add, that the red dots relate to the rivers?

**Reply by hammerdirt-analyst** [reply]
The chart has a legend and the rivers are clearly indicated as the red dots. They are both in the same chart so you can see the difference.

Other changes were made to the requested documents and any other applicable documents in #41

**Comment by ANSBAFU** [comment]
7.1.3. (also correct the other lake chapters accordingly: 8.1.3, ...)
--> change "im" to "am"
Links: Zusammenfassung der Daten aller Erhebungen **am** Bielersee. Rechts: Gefundene Materialarten **am** Bielersee und Prozentsatz deren Gesamtmenge.

**Comment by AmandaFinger** [comment]
There is one thing that our Italian colleague has noticed – please integrate that correctly also in the English and German version of the report (incl. maps + charts):
The “Cerisio” is written correctly as follows: Cer**e**sio (not with an “i”)
The name of the Ceresio is further translated as follows:
German: Luganersee
French: Lac de Lugano
Italian: Ceresio or Lago di Lugano
English: Lake Lugano

**Comment by ANSBAFU** [comment]
German Version
1.1.
--> no "n" in "Flusseinzugsgebieten"
Die Erhebungsgebiete sind nach Flusseinzugsgebiete**n** gruppiert.

**Comment by ANSBAFU** [comment]
German Version
1.1
--> add the words: der Fläche, die
% **der Fläche, die** auf Gebäude entfallen

**Comment by ANSBAFU** [comment]
German Version
1.1
--> replace "intersects" with "Flussmündungen" 
**Intersects** ist ebenfalls eine ordinale Rangfolge der Anzahl der Flüsse/Kanäle, die einen See innerhalb von 1500 m um den Erhebungsort herum durchqueren.

**Comment by ANSBAFU** [comment]
German Version
1.3
--> Description fig. 1.3: change "Dimensionsergebnisse" with "Ergebnisse"
Summen der **Dimensionsergebnisse** für alle Erhebungsgebiete.

**Comment by ANSBAFU** [comment]
German Version 
Abbildung 1.1
--> the meaning of the description of fig. 1.1 is no correct. Please change the second sentence into the following:
**Die Grösse der Markierung entspricht dem Medianwert gefundener Abfallobjekte an diesem Ort.**

**Comment by ANSBAFU** [comment]
German Version
1.1
--> Please change in the last sentence "der Anzahl der Proben" into "der identifizierten Abfallobjekte"
Das Verhältnis **der Anzahl der Proben** bei unterschiedlichen Landnutzungsprofilen gibt einen Hinweis auf die ökologischen und wirtschaftlichen Bedingungen der Untersuchungsstandorte.

**Comment by ANSBAFU** [comment]
German Version
Content table 1.6
--> please change "Fleisgewässer" into "Fliessgewässer"

**Comment by ANSBAFU** [comment]
German Version
Abbildung 1.8
--> please change in fig. 1.8 "Monatsdurchschnitt" into "Monatsdurchschnitt total"

**Comment by ANSBAFU** [comment]
German Version
1.4
--> please integrate "Das" into the link
--> please change "and" to "und" between the two links
Die Erhebungsergebnisse in Bezug auf die Landnutzung auf nationaler Ebene werden ausführlich in **Das** [Landnutzungsprofil] **and** [Geteilte Verantwortung].

**Comment by ANSBAFU** [comment]
German Version
1.5 (please also change in english version)
--> bullet point "Verpackungen ausser Lebensmittel und Getränke": please change into "Verpackungen ausser Lebensmittel, Getränke und Tabakwaren"

**Comment by ANSBAFU** [comment]
German Version
Sentence above Abbildung 1.9 
--> please add the following sentence as in the english version: **Fragmentierte Objekte, die nicht eindeutig identifiziert werden können, werden nach ihrer Größe klassifiziert:**

**Reply by AmandaFinger** [reply]
Make sure not to use the "ß" for double "ss": **Fragmentierte Objekte, die nicht eindeutig identifiziert werden können, werden nach ihrer Grösse klassifiziert:**

**Comment by ANSBAFU** [comment]
German Version
1.6 
--> please change titel "Fleisgewässer" into "Fliessgewässer"

**Comment by ANSBAFU** [comment]
German Version
Abbildung 1.11
--> please change description "Fleisgewässer" into "Fliessgewässer"
--> please adapt the table accordingly to the text

**Comment by ANSBAFU** [comment]
German Version
Abbildung 1.12
--> please change "Fleisgewässer" in the description into "Fliessgewässer"
--> please change it in the entire document (I will not repeat it)

**Reply by AmandaFinger** [reply]
I honestly don't understand why this typo has again appeared in the report, since I have already reported this mistake (e-mail from 31.03.22). It seems that some of your corrections that you have made previously went lost. This kind of work is very time-consuming and tedious for us.

**Comment by ANSBAFU** [comment]
1.7
--> please change titel from "Annex" to "Anhang"

**Comment by ANSBAFU** [comment]
1.1. Land use profile
As much as I understand it is not just about "lakes" but "lakes and rivers" (see below)
Streets are reported as the total number of kilometers of streets within the 1500m radius. Intersects are an ordinal ranking of the number of rivers/canals that intersect a **lake** within 1500m of the survey location.

**Reply by hammerdirt-analyst** [reply]
Hi,
I do not understand the comment. Did you want to change the defintion?

**Reply by ANSBAFU** [reply]
As the chapter is about "lakes and rivers" I though maybe a river also intersects a river. But maybe this wasn't the case in your surveys. Whatever is correct.

**Comment by ANSBAFU** [comment]
@hammerdirt 
The Translation of "std" (standard deviation) in the german version is "Standardfehler" which means "standard error". I guess the correct translation would be "Standardabweichung"! Am I right?

**Reply by hammerdirt-analyst** [reply]
Hi, this is the translation of the definition of "Standard deviation"

_In [statistics](https://en.wikipedia.org/wiki/Statistics), the standard deviation is a measure of the amount of variation or [dispersion](https://en.wikipedia.org/wiki/Statistical_dispersion) of a set of values._

In der Statistik ist die Standardabweichung ein Maß für die Schwankungsbreite oder Streuung einer Reihe von Werten.

**Reply by hammerdirt-analyst** [reply]
Not user how we ended up Standardfehler!

**Comment by ANSBAFU** [comment]
Chapter Seen und Fliessgewässer - Text below "Abbildung 1.1"
 Die Ergebnisse **des Alle** Erhebungsgebiete umfassen 143 Orte, 77 Gemeinden und eine Gesamtbevölkerung von etwa 1 735 991 Einwohnenden. 
Please replace "des Alle" with "der" --> Die Ergebnisse der Erhebungsgebiete....

**Reply by hammerdirt-analyst** [reply]
The issue is raised #48

**Comment by ANSBAFU** [comment]
Chapter Alpen und Jura - Text below 2.2 "Kumulierte Gesamtzahlen für das Erhebungsgebiet
 Die Ergebnisse **des Alpen und Jura** umfassen 20 Orte, 18 Gemeinden und eine Gesamtbevölkerung von etwa 70 606 Einwohnenden.
Please replace "des Alpen und Jura" with "der Alpen und des Juras"

**Reply by hammerdirt-analyst** [reply]
The issue is raised #48

**Comment by ANSBAFU** [comment]
Chapter Aare - Text below 3.1 "Erhebungsorte und Landnutzungsprofile"
Die Ergebnisse des **Erhebungsgebiet** Aare umfassen 51 Orte, 35 Gemeinden und eine Gesamtbevölkerung von etwa 493 799 Einwohnenden.
Please add a "s" to "Erhebungsgebiet". It should be "Erhebungsgebiets"

**Reply by hammerdirt-analyst** [reply]
The issue is raised #47

**Comment by ANSBAFU** [comment]
Chapter Bielersee - Text below 7.1 "Erhebungsorte"
Die Ergebnisse des Bielersee umfassen 10 Orte, 8 Gemeinden und eine Gesamtbevölkerung von etwa 70 810 Einwohnenden.
Please add a "s" to "Bielersee". It should be "Bielersees".

**Reply by hammerdirt-analyst** [reply]
The issue is raised #46

**Comment by ANSBAFU** [comment]
The two last comments would be "valid" for the corresponding chapters

**Reply by hammerdirt-analyst** [reply]
When the revisions are apllied this string will be updated. Thanks.
